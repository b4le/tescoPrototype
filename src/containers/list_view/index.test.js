import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import { TestListView } from './index.jsx';
import renderer from 'react-test-renderer';

describe('List View Component', () => {
    it('renders correctly', () => {
        const lists = [
            {
                "id": "1",
                "title": "First List"
            },
            {
                "id": "2",
                "title": "Second List"
            },
            {
                "id": "3",
                "title": "Third List"
            }
        ]
        const listView = renderer.create(<MemoryRouter><TestListView render='false' lists={lists}/></MemoryRouter>).toJSON();
        expect(listView).toMatchSnapshot();
    });
});