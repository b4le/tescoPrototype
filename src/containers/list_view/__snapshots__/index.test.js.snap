// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page Component renders correctly 1`] = `
<ul
  className="sc-bwzfXH gpaKPE"
>
  <a
    href="/list/1"
    onClick={[Function]}
  >
    <li
      className="sc-bdVaJa fNjGPY"
    >
      First List
    </li>
  </a>
  <a
    href="/list/2"
    onClick={[Function]}
  >
    <li
      className="sc-bdVaJa fNjGPY"
    >
      Second List
    </li>
  </a>
  <a
    href="/list/3"
    onClick={[Function]}
  >
    <li
      className="sc-bdVaJa fNjGPY"
    >
      Third List
    </li>
  </a>
</ul>
`;
